rel/avatar.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "Link Relation: avatar"
+title: "WebFinger Link Relation: avatar"
```

Comment:
I saw it more from the search engine perspective and not as a "namespace" thingy ;)